proto/rpc.proto

```diff
@@ -3,27 +3,31 @@ syntax = "proto3";
 import "keys.proto";
 
 message Invite {
-  // Project info that is displayed to the user receiving the invite
-  message ProjectInfo {
-    optional string name = 1;
-  }
-  bytes projectKey = 1;
-  EncryptionKeys encryptionKeys = 2;
-  optional ProjectInfo projectInfo = 3;
-  string roleName = 4;
+  bytes inviteId = 1;
+  string projectPublicId = 2;
+  string projectName = 3;
+  optional string roleName = 4;
   optional string roleDescription = 5;
-  string invitorName = 6;
+  optional string invitorName = 6;
 }
+
 message InviteResponse {
   enum Decision {
     REJECT = 0;
     ACCEPT = 1;
     ALREADY = 2;
   }
-  bytes projectKey = 1;
+  bytes inviteId = 1;
   Decision decision = 2;
 }
 
+message ProjectJoinDetails {
+  bytes inviteId = 1;
+  bytes projectKey = 2;
+  EncryptionKeys encryptionKeys = 3;
+  string projectName = 4;
```

Comment:
Not sure. I assumed that the invitee would want to see the project name. If they don't, we can omit it here.

Reply:
I think it's necessary in the invite, which is what the invitee sees when they make a decision whether to join, but I hadn't thought there is any UI associated with receiving ProjectJoinDetails?

Reply:
Ah, sorry, I thought this was a comment for another part of the proto.

I don't think there's any UI associated with receiving `ProjectJoinDetails`. This was here in case the project's name changed after the initial invite was sent. (But maybe we don't want that?)

Reply:
Removed in f7e6864a45fad76dc142c367e96f63e2703878e8.